src/middlewared/middlewared/plugins/cache.py

```diff
@@ -1,81 +1,58 @@
-from middlewared.schema import Any, Str, Int, accepts
-from middlewared.service import Service, private
-
 from collections import namedtuple
-import time
+from collections.abc import Callable
+from time import monotonic
+from typing import Any
 
+from middlewared.service import Service
 
-class CacheService(Service):
 
+class CacheService(Service):
     class Config:
         private = True
 
     def __init__(self, *args, **kwargs):
         super(CacheService, self).__init__(*args, **kwargs)
         self.__cache = {}
-        self.kv_tuple = namedtuple('Cache', ['value', 'timeout'])
+        self.kv_tuple = namedtuple("Cache", ["value", "timeout"])
```

Comment:
Why are we changing quotes here?

Reply:
Just standardized the file to double-quotes